apps/navstar_dns/priv/axfr_records

```diff
@@ -1,41 +1,41 @@
-[{dns_rr, <<"_framework._tcp.marathon.mesos.thisdcos.directory">>, 1,
-            33, 5,
-            {dns_rrdata_srv, 0, 0, 36241,
-                <<"marathon.mesos.thisdcos.directory">>}},
-            {dns_rr, <<"_leader._tcp.mesos.thisdcos.directory">>, 1, 33, 5,
-                {dns_rrdata_srv, 0, 0, 5050,
-                    <<"leader.mesos.thisdcos.directory">>}},
-            {dns_rr, <<"_leader._udp.mesos.thisdcos.directory">>, 1, 33, 5,
-                {dns_rrdata_srv, 0, 0, 5050,
-                    <<"leader.mesos.thisdcos.directory">>}},
-            {dns_rr, <<"_slave._tcp.mesos.thisdcos.directory">>, 1, 33, 5,
-                {dns_rrdata_srv, 0, 0, 5051,
-                    <<"slave.mesos.thisdcos.directory">>}},
-            {dns_rr, <<"leader.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 6, 47}}},
-            {dns_rr, <<"marathon.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 6, 47}}},
-            {dns_rr, <<"master.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 6, 47}}},
-            {dns_rr, <<"master0.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 6, 47}}},
-            {dns_rr, <<"mesos.thisdcos.directory">>, 1, 2, 3600,
-                {dns_rrdata_ns, <<"ns.spartan">>}},
-            {dns_rr, <<"mesos.thisdcos.directory">>, 1, 6, 3600,
-                {dns_rrdata_soa, <<"ns.spartan">>,
-                    <<"support.mesosphere.com">>, 1, 60, 180, 86400,
-                    1}},
-            {dns_rr, <<"root.ns1.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 6, 47}}},
-            {dns_rr, <<"root.ns1.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {172, 17, 0, 1}}},
-            {dns_rr, <<"root.ns1.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {198, 51, 100, 1}}},
-            {dns_rr, <<"root.ns1.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {198, 51, 100, 2}}},
-            {dns_rr, <<"root.ns1.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {198, 51, 100, 3}}},
-            {dns_rr, <<"slave.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 3, 101}}},
-            {dns_rr, <<"slave.mesos.thisdcos.directory">>, 1, 1, 5,
-                {dns_rrdata_a, {10, 0, 5, 155}}}].
\ No newline at end of file
+[{dns_rr,<<"_framework._tcp.marathon.mesos.thisdcos.directory">>,1,
```

Comment:
Thanks for this formatting. Mind breaking the reformatting of these files into their own commit?